### ISSUE TITLE
Particles reset/bunch along right side of browser upon viewport resize

### ISSUE THREAD

**Issue by mferrini** [tsparticles/tsparticles#1587]
**Which version are you using?**
27.0

**Is the latest version affected?**

*You can checkout latest version [here](https://www.npmjs.com/package/tsparticles)*

- [x] Yes
- [ ] No

**Which library are you using?**

- [ ] Vanilla JS (`tsparticles`)
- [ ] React (`react-tsparticles`)
- [ ] Vue.js 2.x (`particles.vue`)
- [ ] Vue.js 3.x (`particles.vue3`)
- [ ] Angular (`ng-particles`)
- [ ] Svelte (`svelte-particles`)
- [ ] Preact (`preact-particles`)
- [ ] Inferno (`inferno-particles`)
- [x] jQuery (`jquery-particles`)

<details>
 <summary><b>tsParticles Configuration</b></summary>
  
  ```javascript
  /* $('#page').attr( 'id', 'tsparticles');

  $("#tsparticles")
  .particles()
  .init(
    {
      background: {
        color: {
          value: "transparent",
        },
      },
      fpsLimit: 60,
      interactivity: {
        detectsOn: "canvas",
        events: {
          onClick: {
            enable: false,
            mode: "push",
          },
          onHover: {
            enable: false,
            mode: "repulse",
          },
          resize: true,
        },
        modes: {
          bubble: {
            distance: 400,
            duration: 2,
            opacity: 0.8,
            size: 40,
          },
          push: {
            quantity: 4,
          },
          repulse: {
            distance: 200,
            duration: 0.4,
          },
        },
      },
      particles: {
        color: {
          value: ["#400216","#630321"]
        },
        links: {
          color: "#630321",
          distance: 150,
          enable: true,
          opacity: 0.8,
          width: 1,
        },
        collisions: {
          enable: true,
        },
        move: {
          direction: "none",
          enable: true,
          outMode: "bounce",
          random: false,
          speed: 1,
          straight: false,
        },
        number: {
          density: {
            enable: true,
            value_area: 800,
          },
          value: 80,
        },
        opacity: {
          value: 0.5,
        },
        shape: {
          type: "circle",
        },
        size: {
          random: true,
          value: 6,
        },
      },
      detectRetina: true,
    },
    function (container) {
      // container is the particles container where you can play/pause or stop/start.
      // the container is already started, you don't need to start it manually.
    }
  );
// or

$("#tsparticles")
  .particles()
  .ajax("particles.json", function (container) {
    // container is the particles container where you can play/pause or stop/start.
    // the container is already started, you don't need to start it manually.
  }); */
  ```
</details>

**Describe the bug**
Particles "reset" and bunch along the right side of the screen when viewport is resized.

**To Reproduce**
1. Go to 'https://gerry-tsoukalas.squarespace.com/' Pwd: wharton
2. Click on 'Expand window button.'
3. Or resize browser window by dragging

**Expected behavior**
Particles should expand in synch with viewport width change, instead of bunching along the right side.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [OSX 10.15.7]
 - Browser [Chrome, Safari, Firefox]
 - Version [latest versions]

**Smartphone (please complete the following information):**
Not Applicable: desktop only issue.

**Additional context**
Seems to only happen upon resize/expand.
![Screen Shot 2021-06-02 at 2 06 02 PM](https://user-images.githubusercontent.com/1672614/120552382-0627e080-c3ac-11eb-9a16-532023cf7200.png)

**Comment by matteobruni** [comment]
I reproduced the bug, thanks for the details.

I need to do some checks on the resize function, it will be fixed in the upcoming `1.28.0` version.

**Comment by mferrini** [comment]
Sure thing. 

It's obviously not super crucial to usability or anything, but this client is complaining that when they resize, the particles bunch up, so I figured it was worth logging it as a bug of sorts. 

Other than that your version of particles is very cool, more versatile and fixed the similar issue the original had wherein the particles would bunch weirdly upon any viewport resize, but more to the left, which you can see if you view [https://vincentgarreau.com/particles.js/](https://vincentgarreau.com/particles.js/) with no interactivity configured. 

Anyhow, thanks for taking a look. Much appreciated!

**Comment by matteobruni** [comment]
I released `1.28.0`, it should resize better now, the resize event it's debounced with a delay of 500ms
